library/persistence.js

```diff
@@ -89,6 +91,18 @@ class MongoUserRepository {
       { $set: { current_session_playtime: 0 } })
   }
 
+  async resetDailyTimes (hour) {
+    // first set any non-existing reset hour to 0
```

Comment:
I think I would prefer it if we just didn't update users who don't have a reset hour configured.